src/main/java/org/cryptomator/ui/mainwindow/VaultDetailUnlockedController.java

```diff
@@ -54,6 +98,33 @@ public void showVaultStatistics() {
 		vaultStats.getUnchecked(vault.get()).showVaultStatisticsWindow();
 	}
 
+	private boolean containsUnlockedVaultContent(Path path) {
+		return path.startsWith(vault.get().getAccessPoint());
+	}
+
+	private Optional<Path> getCiphertextPath(Path path) {
+		if (!containsUnlockedVaultContent(path)) {
+			LOG.debug("Path does not start with access point of selected vault: {}", path);
+			return Optional.empty();
+		}
+		try {
+			var cleartextPathString = path.toString().substring(vault.get().getAccessPoint().length());
+			if (!cleartextPathString.startsWith("/")) {
```

Comment:
This hack does not work under windows.

Reply:
I suggest to just make the path relative here and let the method `Vault::getCiphertext` only accept relative paths.

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/2592/commits/c97dbdd169f2de8baaed1dd3686494f69889d954